posts/azure-seen-by-developers.md

```diff
@@ -0,0 +1,190 @@
+---
+authors:
+- Alban Versaveau
+tags:
+- Azure
+- Cloud
+- Serverless
+- Development
+date: 2021-03-22T20:00:00.000Z
+title: "Microsoft Azure, as seen by developers"
+image: https://github.com/aversaveau/blog-usa/blob/azureSeenByDevs/images/2021/03/azure_seen_by_devs_cover.jpg
+---
+
+This article is intended for developers who are new to the Microsoft Azure cloud. It is intended to help them get to grips with the platform in order to help them become operational to create and maintain web applications in Azure. Thus, this article will present the most useful components for developers such as the essential web applications (web apps), databases (DBs), storage spaces (storages) and functions (functions).
+
+Link to the Microsoft Azure platform: [https://portal.azure.com](https://portal.azure.com)
+
+Link to the Azure SDK: [https://azure.microsoft.com/fr-fr/downloads/](https://azure.microsoft.com/fr-fr/downloads/)
+
+# Global view
+
+![Figure 1: Home page](https://github.com/aversaveau/blog-usa/blob/azureSeenByDevs/images/2021/03/azure_seen_by_devs_1.png)
+
+From the home page, you can access the most useful links in the Favorites menu located in the left column. This menu is divided as follows:
+ - All resources
+ - Resource groups
+ - App services
+ - Storage accounts
+ - SQL databases
+ - Azure Database for MySQL
+ - Azure Cosmos DB
+ - Function App
+
+You will now go through these shortcuts that are proposed to you both in terms of utility and interfaces.
+
+## All resources
+
+![Figure 2 : "All resources" view](https://github.com/aversaveau/blog-usa/blob/azureSeenByDevs/images/2021/03/azure_seen_by_devs_2.png)
+
+As its name suggests, this view lists all the resources (web apps, DBs, storages, functions, etc.) to which you have access, for all projects and environments. It is possible to play with the filters as shown in _Figure 2_ to display, for example, all the resources of the integration environment (int) or to display only the web apps of the int.
```

Comment:
The integration environment.
I have updated it as follows :
"...in the integration environment (INT) or to display only the web apps of the INT."
instead of 
"...in the integration environment (int) or to display only the web apps of the int."